manifests/config.pp

```diff
@@ -228,6 +229,25 @@
         content => template('foreman/settings-external-auth.yaml.erb'),
         order   => '02',
       }
+
+      file { "/etc/httpd/conf.modules.d/auth_basic.load":
```

Comment:
I'd really like to avoid this. I created [apache::mod::auth_basic](https://github.com/puppetlabs/puppetlabs-apache/blob/main/manifests/mod/auth_basic.pp) so can we use that instead? So `include apache::mod__auth_basic`.

Reply:
Yeah, I guess that's the right approach, I just did know how to write it in the Puppet way :/